src/EnergyPlus/SingleDuct.cc

```diff
@@ -738,12 +738,6 @@ namespace SingleDuct {
                                 "System",
                                 "Average",
                                 sd_airterminal(SysNum).SysName);
-            SetupOutputVariable("Zone Air Terminal Outdoor Air Volume Flow Rate",
-                                OutputProcessor::Unit::m3_s,
-                                sd_airterminal(SysNum).OutdoorAirFlowRate,
-                                "System",
-                                "Average",
-                                sd_airterminal(SysNum).SysName);
```

Comment:
Moved to a common place such that this variable is setup for all single duct air terminal in in SingleDuct.cc module.